lib/iris/fileformats/pp.py

```diff
@@ -2414,6 +2422,11 @@ def save_pairs_from_cube(cube, field_coords=None, target=None):
         pp_field.lbcode = 1     # Grid code.
         pp_field.bmks = 1.0     # Some scaley thing.
         pp_field.lbproc = 0
+        # Set default MDI value.
+        # Note: *we no longer respect any fill_value of masked data arrays*.
```

Comment:
This "documentation" is going to age really poorly. Is it necessary? Does it not apply to all of these attributes?